internal/common/compress/list.go

```diff
@@ -20,6 +20,14 @@ func CompressStringArray(input []string, compressor Compressor) ([]byte, error)
 	return compressor.Compress(bs)
 }
 
+func MustCompressStringArray(input []string, compressor Compressor) []byte {
```

Comment:
Possibly add some tests to these functions?

Reply:
done